### PR TITLE
Jenkins deploy

### PR DESCRIPTION
**Pull request by blisteringherb**
#### What's this PR do?
I've updated the deploy code to use Jenkins for deploys instead of Wercker.

#### How should this be manually tested?
https://jenkins.dosomething.org/blue/organizations/jenkins/Gladiator%20Deploy/detail/Gladiator%20Deploy/20/pipeline

### CODE REVIEW

File: lib/capistrano/tasks/laravel.rake [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        execute :php, "artisan migrate --force && php artisan cache:clear"
+      end
+    end
+  end
+end
+
+namespace :deploy do
+ after :updated, "laravel:gulp"
+ after :updated, "laravel:artisan_tasks"
+end
```
Comment (sbsmith86) [inline lib/capistrano/tasks/laravel.rake]: newline 😬

**Comment by sbsmith86** [comment]
looks good go to me :)

File: Capfile [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+
+# Includes default deployment tasks
+require 'capistrano/deploy'
+
+# Run Composer and NPM install tasks
+require 'capistrano/composer'
+require 'capistrano/npm'
+
+# Loads custom tasks from `lib/capistrano/tasks' if you have any defined.
+Dir.glob('lib/capistrano/tasks/*.rake').each { |r| import r }
```
Comment (DFurnes) [inline Capfile]: This is nitpicky, but does it make more sense to put these in `config/capistrano` or something like that? Seems kinda odd to have a three-level deep folder structure just for this one rake file.

File: config/deploy.rb [hunk 3]
```diff
@@ -1,55 +1,23 @@
...
+end
 
-  task :artisan_cache_clear do
-    run "cd #{release_path} && php artisan cache:clear"
-  end
 
-  task :restart_queue_worker, :on_error => :continue do
-    run "ps -ef | grep 'queue:work' | awk '{print $2}' | xargs sudo kill -9"
-  end
+set :keep_releases, 1
```
Comment (DFurnes) [inline config/deploy.rb]: Not a merge blocker, but do we want to `set :keep_releases 5` per [this issue](https://github.com/DoSomething/devops/issues/82)?

**Pull request merged**